ocs_ci/ocs/cluster.py

```diff
@@ -1526,6 +1526,26 @@ def wait_for_silence_ceph_osd_crash_warning(osd_pod_name, timeout=900):
         return False
 
 
+def get_mon_config_value(key):
+    """
+    Gets the default value of a specific ceph monitor config
+
+    Args:
+        key (str): Configuration key
+
+    Returns:
+        Configuration value
+
+    """
+    ct_pod = pod.get_ceph_tools_pod()
+    mon_dump_dict = ct_pod.exec_ceph_cmd("ceph mon dump")
+    ceph_mon_name = mon_dump_dict["mons"][0]["name"]
```

Comment:
as far as i remember, dump might be empty, it better use `mon_dump_dict.get()`

Reply:
ack